tests/testsuite.src/run_extensions.at

```diff
@@ -124,6 +124,77 @@ AT_CHECK([$COBCRUN_DIRECT ./prog], [0],
 AT_CLEANUP
 
 
+AT_SETUP([GCOS literals with EBCDIC symbols])
+AT_KEYWORDS([extensions gcos ebcdic])
+
+AT_DATA([prog.cob], [
+       IDENTIFICATION   DIVISION.
+       PROGRAM-ID.      prog.
+       DATA             DIVISION.
+       WORKING-STORAGE  SECTION.
+       01 X             PIC X(11).
+       01 T             PIC 9.
+       01 S             USAGE COMP-1 VALUE ZERO.
+       PROCEDURE        DIVISION.
+           MOVE 1 TO T
+           MOVE ""135,151,151"bar"195,194,234"" TO X
+           IF X NOT EQUAL "foobarBAZ" PERFORM SHOW.
+
+           MOVE 2 TO T
+           MOVE "  "135,151,151"bar"195 194"Z" TO X
+           IF X NOT EQUAL "  foobarBAZ" PERFORM SHOW.
+
+           MOVE 3 TO T
+           MOVE ' '135 151,151'bar'195,194'Z' TO X
+           IF X NOT EQUAL " foobarBAZ" PERFORM SHOW.
+
+           *> GCOS7-specific: should give a ! in between square brackets:
+           *> note square brackets need to be doubled due to m4 preprocessing.
+           MOVE 4 TO T
+           MOVE ""75,80,91"" TO X
+           IF X NOT EQUAL "[[!]]" PERFORM SHOW.
+
+           STOP RUN S.
+       SHOW.
+           DISPLAY "EBCDIC LIT " T " FAILED: X = """ X """"
+           MOVE 1 TO S.
+])
+
+AT_CHECK([$COMPILE -febcdic-symbolic-characters -febcdic-table=ebcdic500_latin1 prog.cob], [0], [], [])
+AT_CHECK([$COBCRUN_DIRECT ./prog], [0], [])
+
+AT_CLEANUP
+
+
+AT_SETUP([GCOS literals with EBCDIC symbols])
+AT_KEYWORDS([extensions gcos ebcdic])
+
+AT_DATA([prog.cob], [
```

Comment:
that's a syntax check, please move to the syn_...at file, that contains the other literal checks, then including also the compile without the new flag

Reply:
I created a file `syn_literals.at` and moved as many tests as possible there, otherwise they would have to be moved into `syn_misc.at` again...

Reply:
You likely have thought about that already:
* the new file must be included in tests/Makefile.am
* the "new" file with the old content should have an entry in tests/ChangeLog
* as noted before: the two run tests must stay in run_extensions, but we _may_ have or miss syntax tests for those, in this case these should be moved/added to the new file
* to decrease syn_misc and have more structure the split test source is very good - to decrease the range of this PR - can you please separate that out to a new one which we can fast-inspect and then distribute with the 3.2 final already?

Reply:
Ok, I must admit I didn't really understand at first the reason why some of those tests were in `run_extensions`, but now, it makes much more sense. I will try to work on such a PR tomorrow.